cartoframes/data/observatory/repository/category_repo.py

```diff
@@ -0,0 +1,41 @@
+from cartoframes.data.observatory.repository.repo_client import RepoClient
+
+
+def get_category_repo():
+    return CategoryRepository()
+
+
+class CategoryRepository(object):
+
+    def __init__(self):
+        self.client = RepoClient()
+
+    def get_all(self):
+        return self._to_categories(self.client.get_categories())
+
+    def get_by_id(self, category_id):
+        result = self.client.get_categories('id', category_id)
+
+        if len(result) == 0:
+            return None
+
+        return self._to_category(result[0])
+
+    def get_by_country(self, iso_code):
+        # TODO
```

Comment:
_Adding a comment here but applies to the rest of the PR_

When adding a #TODO or #FIXME, let's create an issue with a minimal description of what to do or fix and a reference to it in the source code, otherwise it's difficult to track them.

Reply:
OK, perfect. I usually create them just for myself, to remember things before creating the PR (sorry I missed this one).

Reply:
Yep, anyway it's fine having TODOs :)